regulations/generator/layers/paragraph_markers.py

```diff
@@ -11,14 +13,15 @@ class to the old (to make it easier to hide). It exists for backwards
     MarkerInfoLayer instead"""
     shorthand = 'paragraph'
     data_source = 'paragraph-markers'
+    TO_STRIP_RE = re.compile(r'[\(\)\.]')
```

Comment:
FWIW, parentheses and periods lose their special meanings inside character classes and don't have to be escaped (only hyphens are special in this context):

```python
re.sub(r'[().]', '', 'hi.gu(y)s')
# 'higuys'
```

Not necessarily asking for a change here--do you think the intention is clearer with escaping?